llvm/utils/TableGen/SearchableTableEmitter.cpp

```diff
@@ -190,15 +191,18 @@ class SearchableTableEmitter {
   void emitGenericTable(const GenericTable &Table, raw_ostream &OS);
   void emitGenericEnum(const GenericEnum &Enum, raw_ostream &OS);
   void emitLookupDeclaration(const GenericTable &Table,
-                             const SearchIndex &Index, raw_ostream &OS);
+                             const SearchIndex &Index, bool ShouldReturnRange,
```

Comment:
At this point I believe all the callers of this and emitLookupFunction pass Index.ReturnRange for the parameter, so it's unneeded?